### PR TITLE
beep: add page

### PR DESCRIPTION
**Pull request by kalebo**
Added a simple explanation of the linux beep command

### CODE REVIEW

File: pages/linux/beep.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Beep
+
+> A utility to beep the pc speaker.
+
+- Send a series of beeps to the speaker at a specified freqency and duration:
+
+`beep -f {{frequency}} -l {{duration}} -r {{repetitions}}`
```
Comment (waldyrious) [inline pages/linux/beep.md]: Is there a simpler example that could be used for the first one? Say, with one or two options at most.

File: pages/linux/beep.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Beep
+
+> A utility to beep the pc speaker.
+
+- Send a beep that repeats:
```
Comment (waldyrious) [inline pages/linux/beep.md]: Just a nitpick: I'd phrase this as something closer to "Sound a beep a given number of times". ("sound", "play" work better than "send" IMO). By the way, how would one just play a single beep? Just `beep`?

File: pages/linux/beep.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`beep -r {{repetitions}}`
+
+- Send a beep at a specified frequency and duration:
+
+`beep -f {{frequency}} -l {{duration}}`
```
Comment (waldyrious) [inline pages/linux/beep.md]: It would be useful to specify what unit the frequency and duration options use.

**Comment by agnivade** [comment]
LGTM

**Comment by waldyrious** [comment]
[This comment](https://github.com/tldr-pages/tldr/pull/1145#discussion_r85891065) was not addressed. Maybe an overlook? Pinging @kalebo.

**Comment by kalebo** [comment]
In the last commit I'm pretty sure I specified it was hertz and milliseconds. I don't know why github shows it as not being addressed.

**Comment by waldyrious** [comment]
@kalebo I see it now, I must have looked at the wrong line -- the overlook was on my side, sorry about that 😄 LGTM.

**Pull request merged**